src/System.Net.Sockets/src/System/Net/Sockets/BaseOverlappedAsyncResult.Unix.cs

```diff
@@ -23,7 +23,7 @@ public BaseOverlappedAsyncResult(Socket socket, Object asyncState, AsyncCallback
             if (NetEventSource.IsEnabled) NetEventSource.Info(this, socket);
         }
 
-        protected void CompletionCallback(int numBytes, SocketError errorCode)
+        internal void CompletionCallback(int numBytes, SocketError errorCode)
```

Comment:
Sounds good to me.  Sorry to have not caught this earlier when you initially made the comment.